card/lib/card/format.rb

```diff
@@ -203,7 +201,9 @@ def process_content_object override_content=nil, opts={}
       content = override_content || render_raw || ''
       content_object = get_content_object content, opts
       content_object.process_each_chunk do |chunk_opts|
-        prepare_nest chunk_opts.merge(opts) { yield }
+        # Feels scary to just remove it but I can't make any sense of the
+        # "yield" and all tests pass without it
+        prepare_nest chunk_opts.merge(opts) #{ yield }
```

Comment:
I think perhaps "process_content_object" used to take a block arg and I got rid of that?